game/scripts/vscripts/items/cave_rune_hasted_doubledamage.lua

```diff
@@ -0,0 +1,26 @@
+LinkLuaModifier("modifier_rune_haste", LUA_MODIDIER_MOTION_NONE)
+LinkLuaModifier("modifier_rune_doubledamage", LUA_MODIDIER_MOTION_NONE)
+
+item_cave_rune_hasted_doubledamage = class(ItemBaseClass)
+
+function item_cave_rune_hasted_doubledamage:GetIntrinsicModifierName()
+
+  return "modifier_rune_haste", "modifier_rune_doubledamage"
+
+end
+
+function item_cave_rune_hasted_doubledamage:OnSpellStart()
+	local caster = self:Getcaster()
+
+	caster:AddNewModifier(caster, self, "modifier_rune_haste", {duration = 30.0})

```

Comment:
Does that adds the particle effects?
if not we have to add them

Reply:
I cannot find the rune modifiers in the game files, so I have no idea if it includes the particles. Any idea where all the game files can be found so I can find more information?

Reply:
particles/generic_gameplay/rune_doubledamage.vpcf
particles/generic_gameplay/rune_haste.vpcf

Reply:
just use the asset browser to search select particles and search for rune_<effect>